### PR TITLE
💫  update-application-forms

### PR DESCRIPTION
**Pull request by ramsharma-prog**
Application forms updates as per the new prototype

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ARG BASE_IMAGE_TAG="b833677ea06f6409f8555b2743f73bcbd05b6eba"
 FROM ghcr.io/communitiesuk/digital-form-builder-dluhc-runner:$BASE_IMAGE_TAG as base
+
```
Comment (gidsg) [inline Dockerfile:3]: can we revert this as just whitespace change?

**Review by gidsg — APPROVED** [review]
LGTM (aside from minor comment) 👍

File: Dockerfile (hunk 1 above)
Comment (ramsharma-prog) [reply Dockerfile:3]: will do

**Review by rayhan15243 — APPROVED** [review]
Have tested this with the prototype as per our many calls. Happy to approve this.

File: form_jsons/public/applicant-information.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  "lists": [],
+  "sections": [
+    {
+      "name": "zjcAIa",
+      "title": "Applicant information"
+    }
+  ],
+  "conditions": [],
+  "fees": [],
+  "outputs": [],
```
Comment (Adam-W1) [inline form_jsons/public/applicant-information.json:88]: i think we need to add our outputs for save per page?

Comment (Adam-W1) [reply form_jsons/public/applicant-information.json:88]: in this format: 
  "outputs": [
    {
      "name": "RFlVwl",
      "title": "Test",
      "type": "savePerPage",
      "outputConfiguration": {
        "savePerPageUrl": "{{application_store_host}}/applications/forms"
      }
    }
  ],

Comment (ramsharma-prog) [reply form_jsons/public/applicant-information.json:88]: I am pulling another branch & add output to it?

**Pull request merged**